birdhouse/vagrant-utils/install-docker.sh

```diff
@@ -97,5 +97,6 @@ LATEST_COMPOSE_VERSION="`git ls-remote https://github.com/docker/compose | grep
 # LATEST_COMPOSE_VERSION=$(curl --silent https://api.github.com/repos/docker/compose/releases/latest | jq .name -r)  # need jq :(
 sudo curl -L "https://github.com/docker/compose/releases/download/$LATEST_COMPOSE_VERSION/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
 sudo chmod +x /usr/local/bin/docker-compose
-sudo curl -L "https://raw.githubusercontent.com/docker/compose/${LATEST_COMPOSE_VERSION}/contrib/completion/bash/docker-compose" -o /etc/bash_completion.d/docker-compose
+# docker-compose v2 removed the completion file, see https://github.com/docker/compose/issues/8550
+#sudo curl -L "https://raw.githubusercontent.com/docker/compose/${LATEST_COMPOSE_VERSION}/contrib/completion/bash/docker-compose" -o /etc/bash_completion.d/docker-compose
```

Comment:
Is there a value in keeping this comment here?

Reply:
I was half-half on this one.  That line will work for docker-compose v1.  Given issue https://github.com/bird-house/birdhouse-deploy/issues/246, we could potentially use v1 for a bit longer.